vita3k/gui/src/game_install_dialog.cpp

```diff
@@ -56,6 +56,7 @@ void draw_game_install_dialog(GuiState &gui, HostState &host) {
     }
 
     static const auto BUTTON_SIZE = ImVec2(60.f, 0.f);
+    static bool delete_game_file = false;
```

Comment:
Please don't make these static inside a method. Make them global. I beg of you!

Reply:
No i keep it here like other is already set

Reply:
It's not hard to move it to global, it's just a copy paste...

Please...